### DISCUSSION TITLE
Deeper Models for Collision avoidance and object following

### DISCUSSION THREAD

**Discussion by HaithemH** [NVIDIA-AI-IOT/jetbot#396] [Q&A]
Hi all,
I would like to ask you about the performanec of deeper models (e.g, resnet50) and the time comsumption and power?
I would like to get involved with someoen who has jetson nano.

**Comment by jaybdub** [comment]
Hi HaithemH,

Thanks for reaching out!

You can find a variety of benchmarks for various CNN models with Jetson nano here:

https://github.com/NVIDIA-AI-IOT/torch2trt#models

ResNet50 runs at 34 FPS (after optimization) on Jetson Nano.  This profiling was done a while ago, so I'm not sure if this has changed in later software releases.

Please let me know if this helps or you have any questions.

Best,
John

**Reply by HaithemH** [reply]
Thank you for reaching out

**Reply by bonifasius-baskoro** [reply]
Hi John, 
Could I ask something little bit out of this topic but I got confused for something.

When we use the collision avoidance notebook, and input the model. When we run the camera.observe
Does it classify the whole thing in cameras (camera picture as input the model) or does it use some object detection algorithm  like (SSD, Fast-RCNN. etc) and combine with the trained CNN model like ResNet that we trained? 

Thank you for the help.

Boni
